test/e2e/e2e-helpers.js

```diff
@@ -87,6 +87,81 @@ function overrideAnimations(client) {
   });
 }
 
+/**
+ * Set up some helper functions to make writing tests nicer
+ */
+function bootstrapHelpers(client) {
+  // Ideally, it'd just be called setValue, but that's taken
+  /**
+   * Clears the current value and if |value| is specified,
+   *  enters the |value| in.
+   */
+  client.fill = function (selector, value) {
+    this.clearValue(selector);
+    if (typeof value !== 'undefined') {
+      this.setValue(selector, value);
+    }
+
+    // For function chaining
+    return this;
+  };
+
+  /**
+   * For chaining selecting dropdowns
+   * Note: This takes the field name, not the whole selector because it just
+   *  uses selectDropdown above
+   */
+  client.selectDropdown = function (name, value) {
+    selectDropdown(this, name, value);
+    return this;
+  };
+
+  /**
+   * Clicks the input specified if the condition evaluates to true.
+   */
+  client.clickIf = function (selector, condition, ...params) {
```

Comment:
Oh, good point. I think something is in the works to upgrade that to 6.0, but for now, that's an excellent point. It's not really necessary for now, so I can take it out.